k8s/netmanager/stage-airqo-platform-frontend.yaml

```diff
@@ -65,7 +65,7 @@ spec:
     - protocol: TCP
       port: 80
       targetPort: 80
-      nodePort: 31104
+      nodePort: 31101
```

Comment:
Switch old netmanager port for staging to the one reflected in the plan docs [here](https://docs.google.com/document/d/1r0wJLxr2jrtjk8oqnv-N5DBNcORYENeWQntDIjwhVLU/edit#heading=h.yyn93hqir91h)